src/components/Container/Container.jsx

```diff
@@ -1,19 +1,12 @@
 import React, { Component } from 'react';
 import './Container.scss';
-import TopNav from '../../components/TopNav/TopNav';
 
 class Container extends Component {
   render() {
     const { children, option } = this.props;
+
     return (
-      <>
-        <TopNav />
-        <div
-          className={option === undefined ? 'container' : `container ${option}`}
-        >
-          {children}
-        </div>
-      </>
+      <div className={`container ${option ? option : ''}`}>{children}</div>
```

Comment:
굿!!
이 부분은 이렇게도 사용할 수 있습니다!
```suggestion
      <div className={`container ${option || ''}`}>{children}</div>
```

Reply:
이렇게도 사용할 수 있군요! 감사합니다!
Container 뿐만 아니라 Keyword 컴포넌트의 동일한 구문도 수정했습니다!

https://github.com/wecode-bootcamp-korea/fullstack1-1st-BaseNote-frontend/pull/19/commits/2c88b4dd2f2faa07b2a7ff3be4dd4a31e30bf3cb